guacamole-ext/src/main/java/org/apache/guacamole/token/StandardTokens.java

```diff
@@ -23,6 +23,10 @@
 import java.util.Date;
 import org.apache.guacamole.net.auth.AuthenticatedUser;
 import org.apache.guacamole.net.auth.Credentials;
+import java.util.Map;
+import java.util.Set;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
```

Comment:
Guessing you had these in here for debugging purposes, but doesn't look like there's any logging code left, so probably best to get rid of them.  Also, these should probably be alphabetized.